client/src/components/Disclaimer.js

```diff
@@ -9,8 +9,26 @@ import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogContentText from '@material-ui/core/DialogContentText';
 import DialogTitle from '@material-ui/core/DialogTitle';
+import { useBlockstack } from 'react-blockstack';
 
 export const Disclaimer = props => {
+  const { userSession } = useBlockstack();
+
+  const disclaimerAnswer = {
+    answerChoice: null,
+  };
+
+  const storeAnswer = answer => {
+    props.setAnswer(answer);
+    disclaimerAnswer.answerChoice = answer;
+
+    try {
+      userSession.putFile(`disclaimer.json`, JSON.stringify(disclaimerAnswer));
+    } catch (error) {
+      console.error(error);
+    }
+  };
```

Comment:
We should try to keep these functions consistently asynchronous across the board as outlined by Blockstack

Reply:
Sorry to be pedantic, but I was thinking we keep things consistent with the Blockstack documentation by doing something like 

```
await userSession.putFile(...).then(res => {
    doSomething(res)
}).catch(err => 
    doSomething(err)
});
```

Just so we keep our styles consistent across components. If you have a reason why that's frivolous though definitely let me know!

Reply:
I recommend the `useFile()` hook from react-blockstack in function components, rather than accessing the blockstack file api directly. It covers the functionality of putFile/getFile/DeleteFile but is reactive.